src/bundle/Resources/views/admin/content_type/field_types.html.twig

```diff
@@ -212,7 +212,10 @@
                         class="ezselection-settings-option-checkbox ez-selection-table-checkbox"
                         {{ is_translation ? 'disabled' : '' }}>
                     {{ form_errors(option) }}
-                    {{ form_widget(option, { 'attr': { 'class': 'form-control ml-1 mb-0'}}) }}
+                    {{ form_widget(option, { 'attr': {
+                        'class': 'form-control ml-1 mb-0',
+                        'data-ezselection-option-id': option.vars.name }})
```

Comment:
I meant more like adding data-attribute only to the parent `ezselection-settings` like `data-last-id={{ max(form.options|keys) }}`.